### DISCUSSION TITLE
Make Engram appear as a layout option in Gnome DE

### DISCUSSION THREAD

**Discussion by rfiga** [binarybottle/engram-layout#38] [Show and tell]
After installing in Linux we can use the command `setxkbmap -layout us -variant engram` to start using our new keymap. However, in Gnome, it does not persist between reboots or reloads of the DE. Moreover, it fails to successfully set Engram as the current layout in Wayland sessions.

A new variant option needs to be entered in `/usr/share/X11/xkb/rules/evdev.xml` to solve these issues:
```
.
.
.
  <layoutList>
    <layout>
      <configItem>
        <name>us</name>
        <!-- Keyboard indicator for English layouts -->
        <shortDescription>en</shortDescription>
        <description>English (US)</description>
        <languageList>
          <iso639Id>eng</iso639Id>
        </languageList>
      </configItem>
      <variantList>
        .
        .
        .
        <variant>
          <configItem>
            <name>engram</name>
            <description>English (Engram)</description>
          </configItem>
        </variant>
        .
        .
        .
```

Engram now shows up and we can persist and use in Gnome X11 and Wayland sessions.
![add dialog](https://user-images.githubusercontent.com/97045533/147988054-561dbcb5-be77-48c4-b1ef-ee64eed12eea.png)
![layout active](https://user-images.githubusercontent.com/97045533/147988063-d4427f7c-c643-4ea3-8fed-e2a5b1845f66.png)

**Comment by binarybottle** [comment]
That's great, @rfiga, thank you!  Would you please submit a PR for this?

**Reply by rfiga** [reply]
OK, I will work on it.
